src/index.js

```diff
@@ -1,17 +1,20 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
-import './index.css';
-import App from './App';
-import * as serviceWorker from './serviceWorker';
+import React from 'react'
+import ReactDOM from 'react-dom'
+import App from './App'
+import * as serviceWorker from './services/serviceWorker'
+import GlobalStyles from './services/GlobalStyles'
+import GlobalFonts from './fonts/fonts'
 
 ReactDOM.render(
   <React.StrictMode>
+    <GlobalStyles />
+    <GlobalFonts />
```

Comment:
Why don't you make one `GlobalStyles` component with your global styles and fonts?